opensource/project/software-req-win.md

```diff
@@ -9,254 +10,68 @@ menu:
-
-3. Select the **Advanced** tab.
-
-4. Click **Environment Variables**.
-
-	The system opens the **Environment Variables dialog** dialog.
-
-5. Locate the **System variables** area and scroll to the **Path**
-   variable.
-
-    ![windows-mingw](images/path_variable.png)
-
-6. Click **Edit** to edit the variable (you can also double-click it).
-
-	The system opens the **Edit System Variable** dialog.
-
-7. Make sure the `Path` includes `C:\TDM-GCC64\bin`
-
-	 ![include gcc](images/include_gcc.png)
-
-	 If you don't see `C:\TDM-GCC64\bin`, add it.
-
-8. Press **OK** to close this dialog.
-
-9. Press **OK** twice to close out of the remaining dialogs.
-
-## Install Go and cross-compile it
-
-In this section, you install the Go language. Then, you build the source so that it can cross-compile for `linux/amd64` architectures.
-
-1. Open [Go Language download](http://golang.org/dl/) page in your browser.
-
-2. Locate and click the latest `.msi` installer.
-
-	The system prompts you to save the file.
-
-3. Run the installer.
-
-	The system opens the **Go Programming Language Setup** dialog.
-
-4. Select all the defaults to install.
-
-5. Press **Finish** to close the installation dialog.
-
-6. Start a command prompt.
-
-7. Change to the Go `src` directory.
-
-		cd c:\Go\src
-
-8. Set the following Go variables
-
-		c:\Go\src> set GOOS=linux
-		c:\Go\src> set GOARCH=amd64
-
-9. Compile the source.
-
-		c:\Go\src> make.bat
-
-	Compiling the source also adds a number of variables to your Windows environment.
-
-## Task 6. Get the Docker repository
-
-In this step, you start a Git `bash` terminal and get the Docker source code
-from GitHub.
-
-1. Locate the **Git Bash** program and start it.
-
-	Recall that **Git Bash** came with the Git for Windows installation.  **Git
-	Bash** just as it sounds allows you to run a Bash terminal on Windows.
-
-	![Git Bash](images/git_bash.png)
-
-2. Change to the root directory.
-
-		$ cd /c/
-
-3. Make a `gopath` directory.
-
-		$ mkdir gopath
-
-4. Go get the `docker/docker` repository.
-
-		$ go.exe get github.com/docker/docker package github.com/docker/docker
-        imports github.com/docker/docker
-        imports github.com/docker/docker: no buildable Go source files in C:\gopath\src\github.com\docker\docker
-
-	In the next steps, you create environment variables for you Go paths.
+To contribute to the Docker project, you need a <a
+href="https://github.com" target="_blank">GitHub account</a>. A free account is
+fine. All the Docker project repositories are public and visible to everyone.
 
-5. Open the **Control Panel** on your system.
+This guide assumes that you have basic familiarity with Git and Github terminology and usage. Refer to [GitHub For Beginners: Don’t Get Scared, Get Started](http://readwrite.com/2013/09/30/understanding-github-a-journey-for-beginners-part-1/) to get up to speed on Github.
 
-6. Choose **System and Security > System**.
+## 3. Install Git for Windows
```

Comment:
Do you think you should punctuate these sentences? I think I can see why you didn't, but I think I would prefer it.... It's not a deal breaker.